src/components/feed-display/FeedItemDisplay.vue

```diff
@@ -7,6 +7,24 @@
 		<ShareItem v-if="showShareMenu" :item-id="item.id" @close="closeShareMenu()" />
 
 		<div class="action-bar">
+			<NcActions v-show="!hideItemNav"
```

Comment:
The nav items should also not be shown in the screen reader mode, where the DisplayItem is used in the list.
```suggestion
			<NcActions v-show="!splitModeOff"
```
The layout info is already in the component, otherwise you should add the prop also in FeedItemDisplayList where FeedItemDisplay is used for the screen reader mode.

Reply:
good suggestion! re-using `splitModeOff`